fsrs/fsrs.py

```diff
@@ -46,6 +44,59 @@
     0.2,
 )
 
+STABILITY_MIN = 0.001
+LOWER_BOUNDS_PARAMETERS = (
+    STABILITY_MIN,
+    STABILITY_MIN,
+    STABILITY_MIN,
+    STABILITY_MIN,
+    1.0,
+    0.1,
+    0.1,
+    0.0,
+    0.0,
+    0.0,
+    0.01,
+    0.1,
+    0.01,
+    0.01,
+    0.01,
+    0.0,
+    1.0,
+    0.0,
+    0.0,
+    0.0,
+    0.1,
+)
```

Comment:
https://github.com/open-spaced-repetition/fsrs-rs/blob/cc4ae769e08f6fa65136b6980233b212dcba08c3/src/parameter_clipper.rs#L41-L63
```suggestion
LOWER_BOUNDS_PARAMETERS = (
    STABILITY_MIN,
    STABILITY_MIN,
    STABILITY_MIN,
    STABILITY_MIN,
    1.0,
    0.001,
    0.001,
    0.001,
    0.0,
    0.0,
    0.001,
    0.001,
    0.001,
    0.001,
    0.0,
    0.0,
    1.0,
    0.0,
    0.0,
    0.0,
    0.1,
)
```